src/embind/embind.js

```diff
@@ -908,6 +908,7 @@ var LibraryEmbind = {
       }`;
 
 #if EMSCRIPTEN_TRACING
+    invokerFnBody += "if(Object.values(arguments).length > 0){console.log('"+humanName+"(' + Object.values(arguments).join(',') + ')');}else{console.log('"+humanName+"()');}\n";
```

Comment:
Please also format this with using `` ` `` and proper white space.

Reply:
Thanks for the feedback, I am working on it